src/main/java/org/moonlightcontroller/managers/models/HelloRequestSender.java

```diff
@@ -0,0 +1,29 @@
+package org.moonlightcontroller.managers.models;
+
+import org.moonlightcontroller.managers.models.messages.ErrorMessage;
+import org.moonlightcontroller.managers.models.messages.IMessage;
+
+public class HelloRequestSender implements IRequestSender {
+
+	private int xid;
+	
+	public HelloRequestSender(int xid) {

```

Comment:
Hello is not a request, it is not being sent from the controller - why do we need such a class?